lib/api/web-element/scoped-element.js

```diff
@@ -213,7 +218,16 @@ class ScopedWebElement {
       node.name = name;
     }
 
-    return this.waitFor(node.deferred.promise, {isRoot});
+    const exported = {};
```

Comment:
But I checked and `runQueuedCommand` method is only used by action commands and the change is expected in all those commands. And the `waitFor` method only modifies the `this.webElement` property so that it is only resolved once we have the command result, and this change was also doing the same thing, the promise will resolve only when we have the result.

But I agree that the commands are quite a few, so it might not be the right time to do the change.